test-projects/angular-iv/eslint.config.js

```diff
@@ -43,6 +43,6 @@ module.exports = tseslint.config(
   },
   {
     files: ['**/*.ts'],
-    extends: [sheriff.configs.all],
+    extends: [...sheriff.configs.all],
```

Comment:
Note the new eslint config required for the additional rule. I'm ready to discuss filename recognition inside the rules to avoid this change.